src/bitmessageqt/networkstatus.py

```diff
@@ -42,102 +50,159 @@ def __init__(self, parent=None):
+                None,
+                QtCore.QCoreApplication.CodecForTr,
+                shared.numberOfMessagesProcessed))
 
     def updateNumberOfBroadcastsProcessed(self):
+        """Update the counter for the number of processed broadcasts"""
         self.updateNumberOfObjectsToBeSynced()
-        self.labelBroadcastCount.setText(_translate(
-            "networkstatus", "Processed %n broadcast message(s).", None, QtCore.QCoreApplication.CodecForTr, shared.numberOfBroadcastsProcessed))
+        self.labelBroadcastCount.setText(
+            _translate(
+                "networkstatus",
+                "Processed %n broadcast message(s).",
+                None,
+                QtCore.QCoreApplication.CodecForTr,
+                shared.numberOfBroadcastsProcessed))
 
     def updateNumberOfPubkeysProcessed(self):
+        """Update the counter for the number of processed pubkeys"""
         self.updateNumberOfObjectsToBeSynced()
-        self.labelPubkeyCount.setText(_translate(
-            "networkstatus", "Processed %n public key(s).", None, QtCore.QCoreApplication.CodecForTr, shared.numberOfPubkeysProcessed))
+        self.labelPubkeyCount.setText(
+            _translate(
+                "networkstatus",
+                "Processed %n public key(s).",
+                None,
+                QtCore.QCoreApplication.CodecForTr,
+                shared.numberOfPubkeysProcessed))
 
     def updateNumberOfBytes(self):
         """
         This function is run every two seconds, so we divide the rate of bytes
         sent and received by 2.
         """
-        self.labelBytesRecvCount.setText(_translate(
-            "networkstatus", "Down: %1/s  Total: %2").arg(self.formatByteRate(network.stats.downloadSpeed()), self.formatBytes(network.stats.receivedBytes())))
-        self.labelBytesSentCount.setText(_translate(
-            "networkstatus", "Up: %1/s  Total: %2").arg(self.formatByteRate(network.stats.uploadSpeed()), self.formatBytes(network.stats.sentBytes())))
+        self.labelBytesRecvCount.setText(
+            _translate(
+                "networkstatus",
+                "Down: %1/s  Total: %2").arg(
+                    self.formatByteRate(network.stats.downloadSpeed()),
+                    self.formatBytes(network.stats.receivedBytes())))
+        self.labelBytesSentCount.setText(
+            _translate(
+                "networkstatus", "Up: %1/s  Total: %2").arg(
+                    self.formatByteRate(network.stats.uploadSpeed()),
+                    self.formatBytes(network.stats.sentBytes())))
 
     def updateNetworkStatusTab(self, outbound, add, destination):
+        """Add or remove an entry to the list of connected peers"""
+        # pylint: disable=too-many-branches,undefined-variable
         if outbound:
             try:
-                c = BMConnectionPool().outboundConnections[destination]
+                BMConnectionPool().outboundConnections[destination]
```

Comment:
You removed the assignments for `c`

Reply:
Well that would explain it. Not sure how that happened, sorry.